### PR TITLE
OBB to class ES6 + Doc

### PR DESCRIPTION
**Pull request by gchoqueux**
And fix little typo

### CODE REVIEW

**Comment by gchoqueux** [comment]
updated

File: src/Renderer/ThreeExtended/OBB.js [hunk 1]
```diff
@@ -1,133 +1,168 @@
...
+        // this is the same as isPointInsideSphere.position
+        const distance = Math.sqrt((x - localSpherePosition.x) * (x - localSpherePosition.x) +
+                               (y - localSpherePosition.y) * (y - localSpherePosition.y));
+
+        return distance < sphere.radius;
+    }
+
+    /**
+     * Compute OBB from extent.
+     * The OBB resulted can be only in the system 'EPSG:3946'.
```
Comment (zarov) [inline src/Renderer/ThreeExtended/OBB.js:130]: Do you mean `4326` ?

**Review by zarov — APPROVED** [review]
LGTM, thanks for reformatting it in `class` and adding some documentation.

File: src/Renderer/ThreeExtended/OBB.js (hunk 1 above)
Comment (gchoqueux) [reply src/Renderer/ThreeExtended/OBB.js:130]: No, the extent in entry is in `4326` but the result is `3946` because it is meant to viewer

**Pull request merged**